src/chromo/models/dpmjetIII.py

```diff
@@ -146,6 +148,15 @@ def _cross_section(self, kin=None, precision=None):
 
         # TODO set more cross-sections
 
+    def set_cross_section_precision(self, precision=None):
+        self._cross_section_precision = precision
+
+    def get_cross_section_precision(self):
```

Comment:
Why don't you simply use setter and getter for jstatb directly? There is no need keep track of a separate variable. The default value of this variable is also a sensitive choice that @HDembinski asked for.

Also I would call it more technical and representing what it is, like `set_hA_AA_glauber_trials`, so people don't want to touch it. 

Since it is a special option, I would prefer the usage pattern to be:
```
model = DPMJETxxx()
model.set_hA_AA_glauber_trials(100000)
#... do something ... 
```

Reply:
Setter and getter are renamed to  `_hA_AA_glauber_trials`. Now they set `dtglgp.jstatb` parameter directly.